templates/commands/render_action_include.go

```diff
@@ -71,15 +74,19 @@ func actionInclude(ctx context.Context, inc *model.Include, sp *stepParams) erro
 			return err
 		}
 
-		absSrc := filepath.Join(sp.templateDir, walkRelPath)
+		// By default, we copy from the template directory. We also support
+		// grabbing files from the destination directory, so we can modify files
+		// that already exist in the destination.
+		fromDir := sp.templateDir
+		absSrc := filepath.Join(fromDir, walkRelPath)
 		absDst := filepath.Join(sp.scratchDir, relDst)
 
-		skip := skipStrs
+		skipNow := maps.Clone(skip)
```

Comment:
Why do we need to clone?

Reply:
We just want to modify the map for the current iteration of the loop, without affecting future iterations.

This supports the case where the spec files is `include`ing multiple paths. We only want to add this file to the blocklist in the case where the current source dir is the template root. If we're also `include`ing from other dirs that are not the template root, they should get the extra blocklist entry.